packages/gatsby/src/commands/clean.js

```diff
@@ -0,0 +1,16 @@
+const fs = require(`fs-extra`)
+const path = require(`path`)
+
+module.exports = async function clean(args) {
+  const { directory, report } = args
+
+  const directories = [`.cache`, `public`, `node_modules`]
```

Comment:
Shouldn't this exclude `node_modules`?
```suggestion
  const directories = [`.cache`, `public`]
```